### PR TITLE
Update some download and homepage links for Google Code projects

### PR DESCRIPTION
**Pull request by cprecioso**
Related to #10461

### CODE REVIEW

File: Casks/quicklook-pfm.rb [hunk 1]
```diff
@@ -2,9 +2,10 @@
   version '1.1'
   sha256 '53308ac3d7f0f8ed02adb2feff282764fdeadb32352b0776caeaec3d6a514333'
 
+  # googlecode.com is the official download host per the vendor homepage
   url "https://quicklook-pfm.googlecode.com/files/quicklook-pfm-#{version}.zip"
   name 'quicklook-pfm'
-  homepage 'https://code.google.com/p/quicklook-pfm/'
+  homepage 'hthttps://github.com/lnxbil/quicklook-pfm'
```
Comment (Amorymeltzer) [inline Casks/quicklook-pfm.rb]: `hthttps` should read `https`

File: Casks/dictunifier.rb [hunk 2]
```diff
@@ -2,9 +2,9 @@
   version '2.1'
   sha256 'ff80b354ebcbe7ddad0e01d64c667e6a026d92f6bac01d380ec009205679f14c'
 
-  url "https://mac-dictionary-kit.googlecode.com/files/DictUnifier-#{version}.zip"
+  url "https://github.com/jjgod/mac-dictionary-kit/releases/download/v#{version}/DictUnifier-#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/dictunifier.rb]: Please add a `appcast 'https://github.com/jjgod/mac-dictionary-kit/releases/.atom'` line under.

**Comment by cprecioso** [comment]
@Amorymeltzer @vitorgalvao solved

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 5333ce6634b6ec74db4b2ed496257ad1d96b9271 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**